app/code/Magento/GoogleGtag/Block/Ga.php

```diff
@@ -147,20 +147,22 @@ public function getOrdersTrackingData(): array
         foreach ($collection->getItems() as $order) {
             foreach ($order->getAllVisibleItems() as $item) {
                 $result['products'][] = [
-                    'item_id' => $this->escapeJsQuote($item->getSku()),
-                    'item_name' =>  $this->escapeJsQuote($item->getName()),
-                    'price' => number_format((float) $item->getPrice(), 2),
-                    'quantity' => (int)$item->getQtyOrdered(),
+                    'item_id' => $this->_escaper->escapeHtmlAttr($item->getSku()),
```

Comment:
Hi @Bashev , wherever you've replaced `$this->escapeJsQuote` with `$this->_escaper->escapeHtmlAttr` is causing an issue and it replaces the white spaces of the string with **`&#x20;.`** So, can you please update it again with **$this->escapeJsQuote**

Reply:
escapeJsQuote is deprecated.

Reply:
So, Instead of `$this->_escaper->escapeHtmlAttr` you can use `$this->_escaper->escapeHtml` as it will not remove the white spaces in between from the string.